### PR TITLE
#161501194 Setup Login Route

### PR DESCRIPTION
**Pull request by Makwe-O**
#### What does this PR do?
Sets up additional routes for users
#### Description of Task to be completed?
Set up login routes
#### What are the relevant pivotal tracker stories?
#161501194

### CODE REVIEW

**Review by ludralph — COMMENTED** [review]
Do not use underscores to begin variable names

File: _build/index.js [hunk 1]
```diff
@@ -20,23 +20,36 @@ var _saleRoutes = require('./server/routes/sale-routes');
 
 var _saleRoutes2 = _interopRequireDefault(_saleRoutes);
 
+var _userRoutes = require('./server/routes/user-routes');
+
+var _userRoutes2 = _interopRequireDefault(_userRoutes);
+
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```
Comment (ludralph) [inline _build/index.js:27]: Your code should be readable. Ensure you use es6 arrow functions here

File: _build/server/controllers/Product.js [hunk 2]
```diff
@@ -11,26 +11,26 @@ var _Product2 = _interopRequireDefault(_Product);
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
 var Product = {
```
Comment (ludralph) [inline _build/server/controllers/Product.js:13]: convert to es6 class

File: _build/server/controllers/User.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+var _jsonwebtoken = require('jsonwebtoken');
```
Comment (ludralph) [inline _build/server/controllers/User.js:7]: do not begin variable names with underscores

File: _build/server/routes/product-routes.js [hunk 4]
```diff
@@ -23,8 +23,10 @@ var _auth2 = _interopRequireDefault(_auth);
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```
Comment (ludralph) [inline _build/server/routes/product-routes.js:23]: I noticed this function used in several places. Can you abstract it into an helper file and call it from other files. Ensure you use es6 arrow functions

File: _build/index.js (hunk 1 above)
Comment (Makwe-O) [reply _build/index.js:27]: I think I made a mistake of pushing my build folder to the git repo.

File: _build/server/routes/product-routes.js [hunk 5]
```diff
@@ -23,8 +23,10 @@ var _auth2 = _interopRequireDefault(_auth);
...
 
 var router = _express2.default.Router();
-router.get('/', _Product2.default.getAll);
-router.get('/:id', _Product2.default.getOne);
-router.post('/', _auth2.default.ensureToken, _validate2.default.emptyValueProduct, _Product2.default.create);
+router.get('/', _Product2.default.getAllProduct);
+router.get('/:id', _Product2.default.getOneProduct);
+router.post('/', _auth2.default.ensureToken, _validate2.default.emptyValueProduct, _Product2.default.createProduct);
```
Comment (ludralph) [inline _build/server/routes/product-routes.js:28]: Do not begin variable names with underscores

File: _build/index.js (hunk 1 above)
Comment (Makwe-O) [reply _build/index.js:27]: Because that's the transpiled es5 version

File: _build/server/routes/user-routes.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+var _express = require('express');
```
Comment (ludralph) [inline _build/server/routes/user-routes.js:7]: do not begin variable names with underscores and apply this in your entire codebase

File: server/controllers/Product.js [hunk 7]
```diff
@@ -1,25 +1,25 @@
 import ProductModel from '../models/Product';
 
 const Product = {
```
Comment (ludralph) [inline server/controllers/Product.js:3]: Refactor into an es6 class

File: server/controllers/User.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
+import jwt from 'jsonwebtoken';
+import UserModel from '../models/User';
+
+const User = {
```
Comment (ludralph) [inline server/controllers/User.js:4]: refactor into an es6 class

**Pull request merged**